index.bs

```diff
@@ -456,16 +463,23 @@ Abstract Operations {#abstract-operations}
 <div algorithm="construct an orientation sensor object">
 
     : input
+    :: |orientation_interface|, an [=interface=] [=identifier=] whose [=inherited interfaces=] contains {{OrientationSensor}}.
     :: |options|, a {{OrientationSensorOptions}} object.
     : output
-    :: A {{Sensor}} object.
-
-    1. Let |sensor_instance| be the result of invoking [=construct a Sensor object=] with |options|.
-    1. If |options|.{{referenceFrame!!dict-member}} is "screen", then:
-       1.  Define [=local coordinate system=] for |sensor_instance|
-           as the [=screen coordinate system=].
-    1. Otherwise, define [=local coordinate system=] for |sensor_instance|
-       as the [=device coordinate system=].
+    :: An {{OrientationSensor}} object.
+
+    1.  Let |allowed| be the result of invoking [=check sensor policy-controlled features=]
+        with {{OrientationSensor}}.
```

Comment:
with  the [=interface=] identified by |orientation_interface|

Reply:
Absolute and Relative orientation sensors take different FP

Reply:
Fix https://github.com/w3c/orientation-sensor/pull/54/commits/239d32a4bf2cd7105808988730b57c410d78cc87